python/modules/SoftBDeepAK8SFProducer.py

```diff
@@ -439,6 +439,8 @@ def setEff(topPt, catName, topEff, filterArray, topSF, topSFerr, topSFfast, topS
                 sfFast_w = self.topWSFMap["DeepW_fastSF"]["values"][sfBinsFast_w]
                 sfFast_wErr = self.topWSFMap["DeepW_fastSF"]["errors"][sfBinsFast_w]
 
+                eff_sum[eff_sum <= 0] = 0.0001
+
```

Comment:
Will this cause an extreme large value in the SF_effective below, which will propagate to the final topSF? I can't find the cancel out .

Reply:
it will be multiplied by this same value later

Reply:
updated PR to move the safety check so its more obvious (and more robust) that it will cancel out later